src/renderer/components/app.scss

```
@@ -50,6 +50,10 @@
   color: white;
 }
 
+*:target {
```

Probably `*` won't needed in that case.

```suggestion
:target {
```

---

I tested and I did need it.